src/main/java/com/google/api/codegen/DiscoveryFragmentGeneratorApi.java

```diff
@@ -69,6 +70,13 @@
           "The list of YAML configuration files for the fragment generator.",
           ImmutableList.<String>of());
 
+  public static final Option<String> AUTH_INSTRUCTIONS_URL =
+      ToolOptions.createOption(
+          String.class,
+          "auth_instructions",
+          "A comma-delimited map of language to auth instructions URL: lang:URL,lang:URL",
```

Comment:
add a literal `,...`at the end to indicate the map can be more than two entries long